homeassistant/components/sensor/__init__.py

```diff
@@ -32,13 +36,17 @@
 SCAN_INTERVAL = timedelta(seconds=30)
 DEVICE_CLASSES = [
     DEVICE_CLASS_BATTERY,  # % of battery that is left
+    DEVICE_CLASS_CURRENT,  # current (A)
+    DEVICE_CLASS_ENERGY,  # energy (kWh, Wh)
     DEVICE_CLASS_HUMIDITY,  # % of humidity in the air
     DEVICE_CLASS_ILLUMINANCE,  # current light level (lx/lm)
     DEVICE_CLASS_SIGNAL_STRENGTH,  # signal strength (dB/dBm)
     DEVICE_CLASS_TEMPERATURE,  # temperature (C/F)
     DEVICE_CLASS_TIMESTAMP,  # timestamp (ISO8601)
     DEVICE_CLASS_PRESSURE,  # pressure (hPa/mbar)
     DEVICE_CLASS_POWER,  # power (W/kW)
+    DEVICE_CLASS_POWER_FACTOR,  # power factor,cos(ϕ),ϕ (pure number/angle)
```

Comment:
Should we somehow clarify that the power factor doesn't have a unit?

Reply:
It's hard to fully clarify in few word :) Maybe we can insert an inline ref to Wikipedia https://en.wikipedia.org/wiki/Power_factor

Reply:
I'd write eg.
```suggestion
    DEVICE_CLASS_POWER_FACTOR,  # power factor, (no unit, min: -1.0, max: 1.0)
```

Reply:
I've updated the also the docs